express_relay/programs/express_relay/tests/test_set_admin.rs

```diff
@@ -0,0 +1,49 @@
+pub mod helpers;
+
+use anchor_lang::prelude::*;
+use solana_program_test::{ProgramTest, tokio};
+use solana_sdk::{account::Account, signature::Keypair, signer::Signer};
+use express_relay::state::ExpressRelayMetadata;
+use helpers::{initialize, set_admin};
+
+#[tokio::test]
+async fn test_set_admin() {
+    let mut program_test = ProgramTest::new(
```

Comment:
about 50% of this test is the setup. We can make another function that initializes the program and returns all the necessary setup args